module-4/backend/week_one.md

```diff
@@ -4,18 +4,63 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
+
 1. What's the most useful thing you learned from completing the intermission week work?
+
+I enjoyed doing Sorting Suite. I got a lot more comfortable with Javascript/Chai getting to play with it in a familiar context. It helped me to understand a lot of the similarities between Javascript and Ruby, which helps to isolate the differences.
+
 2. What are some tools to help debug JavaScript code?
+
+Debugger is my favorite. There's also console.log and pry
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+We've used Chai so far and are just getting started using Selenium. I'm not comfortabe with Selenium yet, but hopefully I will be by the time I get through Quantified Self.
```

Comment:
Keep in mind that Selenium is less for *unit* tests and more for integration tests.